lesson 17/index.js

```diff
@@ -0,0 +1,91 @@
+"use strict"
+// task 1
+const a = 'true';
+const b = 5 > 6;
+const c = 17;
+const d = undefined;
+let k;
+console.log (a);
+console.log (b);
+console.log (c);
+console.log (d);
+console.log (typeof a, typeof b, typeof c, typeof d, typeof k);
+
+// task 2
+let height = 15;
+let width = 20;
+if (height > width) {
+    console.log (height)
+} else {
+    console.log (width)
+}
+
+// task 3
+for (let i = 1; i < 21; i += 1) {
+    if (i % 3 === 0) console.log (i)
+};
+
+// task 4
+let key = true;
+let documents = true;
+let pen = true;
+let apple = true;
+let orange = false;
+if (key && documents && pen && apple || orange) {
+    alert( 'shouldGoToWork' );
+  }
+
+// task 5
+let n = +prompt ('Введите число', '');
+if (n % 5 === 0, n % 3 !== 0) console.log ('Fiz');
+else if (n % 3 === 0, n % 5 !== 0) console.log ('Buz');
+else if (n % 15 === 0) console.log ('FizBuz');
```

Comment:
Везде рекомендуется использовать фигурные скобки {}
Даже если это одна строка

По реализации это решение некорректное. Вот так лучше
```suggestion
if (n % 5 === 0 && n % 3 === 0) console.log ('FizBuz');
else if (n % 3 === 0) console.log ('Buz');
else if (n % 5 === 0) console.log ('Fiz');
```